asm2vec/binary_to_asm.py

```diff
@@ -50,31 +50,34 @@ def _fn_to_asm(pdf: dict | None, asm_minlen: int) -> str:
     :param asm_minlen: minimum length of assembly functions to be extracted
     :return: ASM string
     """
-    if pdf is None:
-        return ''
-    if len(pdf['ops']) < asm_minlen:
-        return ''
-    if 'invalid' in [op['type'] for op in pdf['ops']]:
-        return ''
-
-    ops = pdf['ops']
-
-    labels, scope = {}, [op['offset'] for op in ops]
-    assert (None not in scope)
-    for i, op in enumerate(ops):
-        if op.get('jump') in scope:
-            labels.setdefault(op.get('jump'), i)
-
-    output = ''
-    for op in ops:
-        if labels.get(op.get('offset')) is not None:
-            output += f'LABEL{labels[op["offset"]]}:\n'
-        if labels.get(op.get('jump')) is not None:
-            output += f' {op["type"]} LABEL{labels[op["jump"]]}\n'
-        else:
-            output += f' {_normalize(op["opcode"])}\n'
-
-    return output
+    try:
+        if pdf is None:
+            return ''
+        if len(pdf['ops']) < asm_minlen:
```

Comment:
If this is the line erroring then:
```suggestion
        if len(pdf.get('ops', []) < asm_minlen:
```
Assuming `pdf['ops']` is a list. 

You can use dict.get to extract from dict and the 2nd arg is a default incase the key is not in the dict